types/index.d.ts

```
@@ -1046,163 +1046,231 @@ export declare function assert(received: any): Assertion;
-  track: () => Function;
-  private recordCall;
-  private recordException;
-  returns(value: any): TrackFn;
-  throws(error: Error): TrackFn;
+	private _implementation;
+	private _calls;
+	private _returnValues;
+	private _exceptions;
+	private _callCount;
+	private _instances;
+	constructor(_implementation: Function);
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)